lib/handlers.js

```diff
@@ -75,6 +75,7 @@ exports.put = function(req, res, next) {
 
   // Update in 1 atomic operation on the database if not specified otherwise
   if (this.shouldUseAtomicUpdate) {
+    this.update_options.new = true;
```

Comment:
From my point of view, I don't see any advantage to get the unmodified document, so I think it's better to force it to true. If you do a PUT request you should get in return the modified object for me.